CITATION.cff

```diff
@@ -0,0 +1,36 @@
+cff-version: 1.2.0
+message: "Please cite this software using these metadata."
+type: software
+title: Shapely
+version: "2.0"
```

Comment:
It's needed if someone uses [GitHub's "Cite this repository" tool](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files). After 2.0 is released, I suggest this not be updated until the next release, so that the GitHub tool is useful for the "current" release (when we are there). Also, this version and release date override git tags for Zenodo, so it's important it is updated before tagging and releasing.